pipeline/generic_pipeline.go

```diff
@@ -1137,6 +1156,118 @@ func (genericPipeline *GenericPipeline) updatePipelineSettings(settings metadata
 	genericPipeline.updateTimestampsSetting(settings)
 	genericPipeline.updateProblematicVBSettings(settings)
 	genericPipeline.updateTopologyProgress(settings)
+	genericPipeline.updateConflictLoggingRules(settings)
+}
+
+func (genericPipeline *GenericPipeline) updateConflictLoggingRules(settings metadata.ReplicationSettingsMap) {
```

Comment:
Live update for conflictLogging setting should be fine since the HLV stamping behaviour is governed by crossClusterVersioning setting instead. I was talking about mobile setting to be made non-live.